### ISSUE TITLE
Update cloud-functions on all environments on "deploy" command

### ISSUE THREAD

**Issue by ericboucher** [aqualinkorg/aqualink-app#527]
To make testing environment more accurate, we want to deploy cloud-functions to all environments

In addition, to avoid any out-of-sync issues between cloud functions and the DB schema, cloud-functions should be updated with "deploy" command.

**Comment by avalmas-programize** [comment]
`Aqualink-Programize` account has not access to `functions` functionality

**Comment by avalmas-programize** [comment]
`Aqualink-dev` has the `functions` available, however the cloud resource location is not set which required for scheduled functions
```
Cloud resource location is not set for this project but scheduled functions requires it. Please see this documentation for more details: https://firebase.google.com/docs/projects/locations.
```

**Comment by ericboucher** [comment]
@avalmas-programize I upgraded the aqualink-programize environment and it should be able to run functions and I set the cloud locations for both apps.

**Comment by avalmas-programize** [comment]
Cloud functions have been deployed on both environments successfully
On `aqualink-programize`  function `scheduledDailyUpdate` failed once with an out memory error

**Comment by ericboucher** [comment]
@avalmas-programize we still need to address the second part of the issue:

`In addition, to avoid any out-of-sync issues between cloud functions and the DB schema, cloud-functions should be updated with "deploy" command.`

**Comment by ericboucher** [comment]
Solved in #558
